211014_SARIF_TC_46.md

```diff
@@ -0,0 +1,186 @@
+
+## 1.3 Procedures for this meeting (Co-Chair David)
+
+## 1.4 Approval of agenda (Co-Chair David)
+
+* https://www.oasis-open.org/committees/download.php/69179/agenda_20211014.html
+
+  Agenda was approved with the following changes:
+
+  * Only discussing subitems 3.1 and 3.8 during meeting.
+
+## 1.5 Approval of previous minutes (Co-Chair David)
+
+* [Minutes of 2021-09-30 Meeting #45](https://www.oasis-open.org/committees/download.php/69178/210930_SARIF_TC_45.txt)
+
+  Minutes were approved as amended per [210930_SARIF_TC_45.txt](https://github.com/oasis-tcs/sarif-spec/blob/master/210930_SARIF_TC_45.txt)
+
+## 1.6 Review of action items and resolutions (Previous Meeting Secretary Michael)
+
+* ACTION Michael/Eddy:  Finalize and review errata for possible improvements
+  * Michael asks for reviewers and guidance
+* ACTION Michael: Update OASIS repo teams/permissions to include other TC members
+  * DONE
+* ACTION Michael: Finalize errata docs and send to David for next steps, ballot, etc. Finalized, but unsent
+  * DONE
+* ACTION Michael: Reach out to MicroFocus on 7PK copyright issues, if any
+  * DONE
+* ACTION Michael: Other outreach to Parasoft, StackHawk, ForAllSecure, NTT/WhiteSource
+  * DONE (residual action transferred to a new action)
+* ACTION David: Author Wikipedia page
+  * ONGOING
+* ACTION Paul: produce more information on metrics, MISRA proposals, etc
+  * ONGOING
+ 
+## 1.7 Identification of SARIF TC voting members (Co-Chair David)
+
+All voting members present.
+
+### 1.7.1 Prospective members attending their first meeting
+
+### 1.7.2 Members attaining voting rights at the end of this meeting
+
+### 1.7.3 Members losing voting rights if they have not joined this meeting by the time it ends
+
+### 1.7.4 Members who previously lost voting rights who are attending this meeting
+
+### 1.7.5 Members who have declared a leave of absence
+
+# 2. Future Meetings
+
+## 2.1 Future meeting schedule (Co-Chair David)
+
+* Scheduled Teleconference (Thursday at 08:00 PDT / 15:00 UTC for 1.5 hours)
+  ```
+  October 28
+  ```
+* Proposed Teleconferences (Tuesdays at 08:00 PDT / 16:00 UTC for 1.5 hours)
+  ```
+  November 9
+  November 23
+  ```
+  Notes
+  1. For November only, Tuesdays are proposed to avoid conflicting with other meetings and with the US Thanksgiving holiday.
+  2. For the proposed November dates, the US and EU will both be off daylight savings time, thus the difference in UTC time.
+
+  Discussion if November 11 and December 2 match better (keeping the Thursdays and avoid Thanksgiving)
+
+  * Next meetings agreed:
+  ```
+  November 11
+  December 2
+  ```
+
+* Possible face-to-face meeting in 2022Q1
+
+# 3. Discussion
+
+## 3.1 Discuss dynamic analysis and how to address it
+
+* What to address
+  * Michael suggests expanding the scope, keeping the brand
+  * Stefan seconds to expand the scope, provide software instead of static coverage, but keep
```

Comment:
keep the brand?

Reply:
By this, I meant, preserve 'SARIF' as an acronym/brand. If we promote DARIF or SARI (pronounced 'sorry'?) for new extensions, we risk creating confusion, Right now, our intent is to continue to build on previous work while maintaining compatibility.

If, in future, we give up on this goal and pursue an entirely distinct format, by design, we would then discuss a new acronym/brand.